app/jobs/in_person/enrollments_ready_for_status_check/uses_sqs_client.rb

```diff
@@ -0,0 +1,48 @@
+module InPerson::EnrollmentsReadyForStatusCheck
+  module UsesSqsClient
```

Comment:
If I'm reading this right, it looks like we are only including this mixin in one place. Do we have plans for future SQS topics? (my understanding is no, we do not) so I think that it would make sense to just inline this for now

Reply:
This mixin is used by the job (included by both the job directly and the batch processor), but I'm intentionally separating this out to reduce the complexity of working on this section of the code. I did consider renaming this to `SqsBatchWrapper`.

Reply:
What about using delegation instead of inheritance via mixin? To have an SQS client/consumer class that we can instantiate instead of mixing in? delegation is much easier to follow than included methods from mixins when tracing code

Reply:
I was considering that exact refactor before posting this PR. If you think it's worth making the change, then I'll do it.

Reply:
yes definitely, let's go for it

Reply:
@zachmargolis I refactored to use DI instead of modules. There's some lack of clarity about where to put the factories, so for now they live in the job and have their own tests.